functions/New-DbaDirectory.ps1

```diff
@@ -76,9 +76,9 @@ function New-DbaDirectory {
 
         $Path = $Path.Replace("'", "''")
 
-        $exists = Test-DbaPath -SqlInstance $SqlInstance -SqlCredential $SqlCredential -Path $Path
+        $exists = Test-DbaPath -SqlInstance $Instance -SqlCredential $SqlCredential -Path $Path
```

Comment:
This should be passing the `$server` object, internal calls to other public functions have the main object passed so it does not cause additional connections to be opened in SQL Server.

```
$exists = Test-DbaPath -SqlInstance $server -Path $Path
```